bindings/core/src/method_handler/utils.rs

```diff
@@ -129,7 +129,11 @@ pub(crate) fn call_utils_method_internal(method: UtilsMethod) -> Result<Response
                 protocol_parameters,
             );
 
-            Response::TransactionFailureReason(context.validate()?)
+            if let Err(e) = context.validate() {
+                Response::TransactionFailureReason(Some(e))
+            } else {
+                Response::TransactionFailureReason(None)
+            }
```

Comment:
Don't we now want to return an error in the bindings as well?

Reply:
So removing ` Response::TransactionFailureReason` and letting `validate` error?

Reply:
Yes, but not sure if that's really better, that's why I made it a question